tagstudio/src/core/media_types.py

```diff
@@ -307,6 +327,12 @@ class MediaCategories:
         ".xls",
         ".xlsx",
     }
+    _SCRIPT_SET: set[str] = {
+        ".com",
+        ".bat",
+        ".ps1",  ##Powershell script
+        ".sh",  ##Bash script
```

Comment:
Consider adding ".bash" and ".zsh". Also see above on sorting and comments, thanks!